pkg/server/node.go

```diff
@@ -2108,8 +2108,16 @@ type lockedMuxStream struct {
 func (s *lockedMuxStream) SendIsThreadSafe() {}
 
 func (s *lockedMuxStream) Send(e *kvpb.MuxRangeFeedEvent) error {
+	// TODO(michael): where should this be defined and what's a reasonable threshold?
+	const slowMuxStreamLockLogThreshold = 0 * time.Minute
+	start := timeutil.Now()
```

Comment:
We have a package called `crtime` that we should use here instead. `crtime` is a bit more efficient for timing calls that might be called in tight loop because it only consults the monotonic clock rather than also needing the wallclock time.

Reply:
Sweet, thanks for pointing that out :)